tests/zenko_tests/node_tests/iam_policies/cloudserver/AssumeRole.js

```diff
@@ -0,0 +1,330 @@
+const assert = require('assert');
+const async = require('async');
+const { errors } = require('arsenal');
+const VaultClient = require('../../VaultClient');
+const { getS3Client } = require('../../s3SDK');
+const { getSTSClient } = require('../../stsSDK');
+const { metadataSearchResponseCode } = require('./utils');
+
+const clientAdmin = VaultClient.getAdminClient();
+
+const account1Name = 'iam-policies-assume-role-test-account1';
+const account1Info = {
+    email: `${account1Name}@test.com`,
+};
+const externalAccessKey1 = 'DZMMJUPWIUK8IWXRP0HQ';
+const externalSecretKey1 = 'iTuJdlidzrLipymvAGrLP66Yxghl4NQxLZR3cLlu';
+
+const account2Name = 'iam-policies-assume-role-test-account2';
+const account2Info = {
+    email: `${account2Name}@test.com`,
+};
+const externalAccessKey2 = 'DZMMJUPWIUK8IWXRP0HR';
+const externalSecretKey2 = 'iTuJdlidzrLipymvAGrLP66Yxghl4NQxLZR3cLlv';
```

Comment:
Same as your other PR, I don't understand why you use hardcoded keys when you can just use the ones returned from the function